doc/classes/InputEventAction.xml

```diff
@@ -14,7 +14,7 @@
 	</tutorials>
 	<members>
 		<member name="action" type="StringName" setter="set_action" getter="get_action" default="&amp;&quot;&quot;">
-			The action's name. Actions are accessed via this [String].
+			The action's name. Actions are accessed via this [StringName].
```

Comment:
Maybe we should avoid the type reference, and instead provide more useful info. CC @Mickeon

Reply:
If you read it slowly you may realise it's genuinely redundant information, yes. The latter sentence should be removed.

Reply:
Well it's a bit of a meme that a property `action: StringName` would have as description `The action's name.`
That's not telling users much.

I think the second sentence adds important information that this name isn't just cosmetic but the actual identifier used in other APIs to reference to this action.

Reply:
What about something like this:

> The action's name. This is usually the name of an existing action in the [InputMap].

Reply:
Yeah maybe:

> The action's name. This is usually the name of an existing action in the [InputMap] which you want this custom event to match.